<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Main Button</string>
    </property>
    <widget class="QPushButton" name="runPB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connectPB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="settingPB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Setting</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitPB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>401</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>MPR121</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>381</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="mprStatLayout"/>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>461</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="MPR_Reg_TB">
     <attribute name="title">
      <string>MPR121</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>451</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string>Register Setting</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QLabel" name="mprMHDRLabel_2">
          <property name="text">
           <string>MHDR(0x2B):</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <widget class="QLabel" name="mprNHADRLabel_2">
          <property name="text">
           <string>NHADR(0x2C):</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_18">
        <item>
         <widget class="QLabel" name="mprNCLRLabel_2">
          <property name="text">
           <string>NCLR(0x2D)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_18">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <item>
         <widget class="QLabel" name="mprNHDAFLabel_2">
          <property name="text">
           <string>NHDAF(0x30)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_19">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_20">
        <item>
         <widget class="QLabel" name="mprMHDFLabel_2">
          <property name="text">
           <string>MHDF(0x2F)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_20">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_21">
        <item>
         <widget class="QLabel" name="mprFDLRLabel_2">
          <property name="text">
           <string>FDLR(0x2E)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_21">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_22">
        <item>
         <widget class="QLabel" name="mprNCLFLabel_2">
          <property name="text">
           <string>NCLF(0x31)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_22">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>230</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_23">
        <item>
         <widget class="QLabel" name="mprCDCFLabel_2">
          <property name="text">
           <string>CDC_FILTER(0x5C)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_23">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>30</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_24">
        <item>
         <widget class="QLabel" name="mprFDLFLabel_2">
          <property name="text">
           <string>FDLF(0x32)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_24">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>150</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_25">
        <item>
         <widget class="QLabel" name="mprFDLTLabel_2">
          <property name="text">
           <string>FDLT(0x35)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_25">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>190</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_26">
        <item>
         <widget class="QLabel" name="mprDEBOUNCELabel_2">
          <property name="text">
           <string>DEBOUNCE(0x5B)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_26">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>70</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_27">
        <item>
         <widget class="QLabel" name="mprNHDATLabel_2">
          <property name="text">
           <string>NHDAT(0x33)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_27">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>110</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_28">
        <item>
         <widget class="QLabel" name="mprNCLTLabel_2">
          <property name="text">
           <string>NCLT(0x34)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_28">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>270</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_29">
        <item>
         <widget class="QLabel" name="mprCDTFLabel_2">
          <property name="text">
           <string>CDT_FILTER(0x5D)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="mprtb_syncPB">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>330</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Sync</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ADS_Reg_TB">
     <attribute name="title">
      <string>ADS1292</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>451</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string>Register Setting</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_29">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_30">
        <item>
         <widget class="QLabel" name="adsCONFIG1Label">
          <property name="text">
           <string>CONFIG1(0x01):</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_30">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_31">
        <item>
         <widget class="QLabel" name="adsCONFIG2Label">
          <property name="text">
           <string>CONFIG2(0x02):</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_31">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_32">
        <item>
         <widget class="QLabel" name="adsLOFFLabel">
          <property name="text">
           <string>LOFF(0x03)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_32">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_33">
        <item>
         <widget class="QLabel" name="adsRLD_SENSLabel">
          <property name="text">
           <string>RLD_SENS(0x06)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_33">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_34">
        <item>
         <widget class="QLabel" name="adsCH2SETLabel">
          <property name="text">
           <string>CH2SET(0x05)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_34">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_35">
        <item>
         <widget class="QLabel" name="adsCH1SETLabel">
          <property name="text">
           <string>CH1SET(0x04)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_35">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_36">
        <item>
         <widget class="QLabel" name="adsLOFF_SENSLabel">
          <property name="text">
           <string>LOFF_SENS(0x07)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_36">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>230</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_37"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_37">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>30</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_38">
        <item>
         <widget class="QLabel" name="adsLOFF_STATLabel">
          <property name="text">
           <string>LOFF_STAT(0x08)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_38">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>150</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_39">
        <item>
         <widget class="QLabel" name="adsGPIOLabel">
          <property name="text">
           <string>GPIO(0x0B)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_39">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>190</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_40"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_40">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>70</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_41">
        <item>
         <widget class="QLabel" name="adsRESP1Label">
          <property name="text">
           <string>RESP1(0x09)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_41">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>110</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_42">
        <item>
         <widget class="QLabel" name="adsRESP2Label">
          <property name="text">
           <string>RESP2(0x0A)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_42">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>270</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_43"/>
      </widget>
     </widget>
     <widget class="QPushButton" name="adstb_syncPB">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>330</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Sync</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>410</y>
      <width>671</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>ADS1292</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>651</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="adsChartLayout"/>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
